forschungsjournal-soziale-bewegungen-fjsb.csl

```diff
@@ -0,0 +1,172 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="de-DE">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Forschungsjournal Soziale Bewegungen FJSB(German)</title>
+    <title-short>FJSB (German)</title-short>
+    <id>http://www.zotero.org/styles/forschungsjournal-soziale-bewegungen-fjsb</id>
+    <link href="http://www.zotero.org/styles/forschungsjournal-soziale-bewegungen-fjsb" rel="self"/>
+    <author>
+      <email>jannis.grimm@fu-berlin.de</email>
+      <name>Jannis Grimm</name>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="linguistics"/>
+    <category field="literature"/>
+    <issn>2365-9890</issn>
+    <summary>FJSB</summary>
+    <updated>2021-11-01T09:59:16+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
+    <terms>
+      <term name="et-al">et al.</term>
+      <term name="editor" form="short">Hg.</term>
+      <term name="online">Online unter</term>
+    </terms>
+  </locale>
+  <macro name="creator-year-long">
+    <group delimiter=" ">
+      <names variable="author" font-style="italic">
+        <name delimiter="/" name-as-sort-order="all"/>
+        <label form="short" prefix=" (" suffix=")"/>
+        <substitute>
+          <names variable="editor"/>
```

Comment:
Typically you'll want either title or an anon macro here. Otherwise you get the weird behavior you see for the webpage citation in the preview

Reply:
I'm sorry, I don't understand what you mean here. My apologies

Reply:
Fixed, I hope